app/views/coronavirus_landing_page/components/shared/_country_section.html.erb

```diff
@@ -1,18 +1,21 @@
-<section class="covid__topic-wrapper">
-  <%= render "govuk_publishing_components/components/heading", {
-    text: country_section["header"],
-    padding: true,
-    border_top: 2
-  } %>
-  <p class="govuk-body-m">
-    <%= country_section["text"] %>
-  </p>
-
-  <ul class="govuk-list">
-    <% country_section["links"].each do | link | %>
-      <li class="covid__topic-list-item">
-        <%= link_to link["label"].html_safe, link["url"], class: 'covid__topic-list-link govuk-link' %>
-      </li>
-    <% end %>
-  </ul>
-</section>
+<%
+  guidance ||= nil
+%>
+<%= 
+  if guidance.present? && guidance["links"].present?
+    guidance_links = guidance["links"]
+    content_tag :p, :class => 'govuk-body' do
+      concat(guidance["text"] + " ")
+      guidance_links.each do | link |
+        elem =  link_to(link["label"],link["url"], class: "govuk-link")
+        if  link.equal? guidance_links.last
+          concat(elem)
+        elsif link.equal? guidance_links[guidance_links.length - 2]
+          concat(elem + " and ")
+        else
+          concat(elem + ", ")
+        end
```

Comment:
This is very impressive and clever, but I wonder if it's too complicated? I don't know if it needs to be this flexible? Could have something simpler like:

```
<p class="govuk-body">
  Additional guidance for
  <-- scotland link -->
  <-- wales link -->
  and
  <-- ni link -->
</p>

Reply:
Hm, yes it is over engineered but we could _theoretically_  take this piece of code and have it render any text we want, followed by any number of links we want in the _[link(1)](), [link(2)](), [link(3)](),..., [link(n-1)]() and [link(n)]()_  format.

Whether we _need_ this level of flexibility right now is, yeah... debatable.